File: GettingStartedWithASPNETCore.VisualBasic/Number.vb
Public Class Number
    Public Shared Function IsPrime(ByVal number As Integer) As Boolean
        Dim primeI As Integer
        Dim primeFlag As Boolean
        primeFlag = True
        For primeI = 2 To number / 2
            If number Mod primeI = 0 Then
                Return False
            End If
        Next
        Return primeFlag
    End Function
End Class

File: Codegen 2021/Getting started with .NET Core/code/GettingStartedWithASPNETCore.Console/Number.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GettingStartedWithASPNETCore.ConsoleApp
{
    public static class Number
    {
        public static bool IsPrime(int number)
        {
            int primeI;
            bool primeFlag;
            primeFlag = true;
            for (primeI = 2; primeI <= number/2; primeI++)
            {
                if (number % primeI == 0)
                {
                    return false;
                }
            }
            return primeFlag;
        }
    }
}